profanity.rb

```diff
@@ -28,7 +28,7 @@
 require 'curses'
 require 'fileutils'
 require 'ostruct'
-include Curses
```

Comment:
you shouldn't need to `include Curses` since this is really for sharing behavior and Curses is a class not a module. using `require` is all that's needed